tests/models/test_semantic_segmentation.py

```diff
@@ -626,25 +634,24 @@ def __init__(self):
 def test_functional_pretrained(remote_sample, tmp_path):
     """Test for load up pretrained and over-writing tile mode ioconfig."""
     save_dir = pathlib.Path(f"{tmp_path}/output")
-    mini_wsi_svs = pathlib.Path(remote_sample("svs-1-small"))
+    mini_wsi_svs = pathlib.Path(remote_sample("wsi4_1k_1k_svs"))
     reader = get_wsireader(mini_wsi_svs)
     thumb = reader.slide_thumbnail(resolution=1.0, units="baseline")
-    thumb = thumb[1024:1536, 1024:1536, :]
```

Comment:
why not use a 512x512 image? Why stick to 1kx1k?

Reply:
I uploaded 512x512.svs and used that now.

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/commit/0e9b817ff3c0ae08d9250f7da6f8a819f422f17a